services-js/registry-certs/client/birth/components/SupportingDocumentsInput.tsx

```diff
@@ -1,22 +1,31 @@
 import React from 'react';
 import { css } from 'emotion';
 
+// todo: prevent next/back while uploads pending
+
 import {
   CloseButton,
+  CHARLES_BLUE,
+  ERROR_TEXT_COLOR,
   FOCUS_INDICATOR_COLOR,
+  OPTIMISTIC_BLUE_LIGHT,
   VISUALLYHIDDEN,
+  MEDIA_SMALL,
+  SERIF,
 } from '@cityofboston/react-fleet';
 
+import AnswerIcon from '../icons/AnswerIcon';
+
+import UploadableFile from '../../models/UploadableFile';
+import { observer } from 'mobx-react';
+
 interface Props {
-  name: string;
-  title: string;
-  fileTypes?: string[] | string;
-  sizeLimit: FileSize;
-  handleChange(documents: File[]): void;
+  uploadSessionId: string;
+  selectedFiles?: UploadableFile[];
```

Comment:
If you require this to always exist, it would allow you to remove some existence checks down below. (I would assume callers would pass it an empty array on first load.)